glib-macros/src/lib.rs

```diff
@@ -885,15 +885,18 @@ pub fn cstr_bytes(item: TokenStream) -> TokenStream {
 /// You might hit a roadblock when declaring properties with this macro because you want to use a name that happens to be a Rust keyword. This may happen with names like `loop`, which is a pretty common name when creating things like animation handlers.
 /// To use those names, you can make use of the raw identifier feature of Rust. Simply prefix the identifier name with `r#` in the struct declaration. Internally, those `r#`s are stripped so you can use its expected name in [`ObjectExt::property`] or within GtkBuilder template files.
 ///
-/// # Generated wrapper methods
+/// # Generated methods
 /// The following methods are generated on the wrapper type specified on `#[properties(wrapper_type = ...)]`:
 /// * `$property()`, when the property is readable
 /// * `set_$property()`, when the property is writable and not construct-only
 /// * `connect_$property_notify()`
 /// * `notify_$property()`
 ///
-/// Notice: You can't reimplement the generated methods on the wrapper type,
-/// but you can change their behavior using a custom internal getter/setter.
+/// You can choose to move the method definitions to a trait by using `#[properties(wrapper_type = MyType, ext_trait = MyTypeExt)]`.
```

Comment:
```suggestion
/// You can choose to move the method definitions to a trait by using `#[properties(wrapper_type = MyType, ext_trait = MyTypeExt)]`.
```
I wonder if we should have a default name for the trait here and allow the user to set a custom one? This way we can default to something like `MyTypePropertiesExt`?